src/DIPS.Xamarin.UI/Internal/Xaml/SheetView.xaml

```diff
@@ -31,20 +33,76 @@
             </Grid.RowDefinitions>
             <Grid x:Name="HeaderGrid"
                   Grid.Row="0"
-                  Padding="0,5,0,5"
+                  RowSpacing="0"
+                  ColumnSpacing="0"
                   BackgroundColor="{Binding HeaderColor}">
-                <BoxView HeightRequest="5"
+
+                <Grid.RowDefinitions>

```

Comment:
You need to update the code behind `Initialize` method to reflect these changes, setting the the sheet to start from Top does not work now.

Reply:
So, to sum up the `Initialize`: It's whats actually is setting up the Grid, this isn't very obvious.
The reason I've done it this way is to not have to write double XAML for both cases. I totally forgot about this and it's not really obvious.

Maybe we can add a internal XAML comment here to remember ourselves this if we mess with the grid.